docs/content/latest/explore/indexes-constraints/indexes-1.md

```diff
@@ -0,0 +1,172 @@
+
+You can apply sort order on the indexed columns as `ASC` (default), `DESC`, as well as `HASH`. For examples, see [HASH and ASC Examples](https://docs.yugabyte.com/latest/api/ysql/the-sql-language/statements/ddl_create_index/#unique-index-with-hash-column-ordering)
+
+You can use the `EXPLAIN` statement to check if a query uses an index.
+
+Suppose you work with a database that includes the following table populated with data:
+
+```sql
+CREATE TABLE employees (
+  employee_no integer,
+  name text,
+  department text
+);
+```
+
+```sql
+INSERT INTO employees VALUES 
+(1221, 'John Smith', 'Marketing'),
+(1222, 'Bette Davis', 'Sales'),
+(1223, 'Lucille Ball', 'Operations'); 
+```
+
+The following example shows a query that finds employees working in Operations:
+
+```sql
+SELECT * FROM employees WHERE department = 'Operations';
+```
+
+To process the preceding query, the whole `employees` table needs to be scanned. For large organizations, this might take significant amount of time.
+
+To speed up the process, you create an index for the department column, as follows:
+
+```sql
+CREATE INDEX index_employees_department
+  ON employees(department);
+```
+
+The following example executes the query after the index has been applied to `department` and uses the `EXPLAIN` statement to prove that the index participated in the processing of the query:
+
+```sql
+EXPLAIN SELECT * FROM employees WHERE department = 'Operations';
+```
+
+The following is the output produced by the preceding example:
+
+```
+QUERY PLAN                        
+-----------------------------------------------------------------------------------
+Index Scan using index_employees_department on employees (cost=0.00..5.22 rows=10 width=68)
+Index Cond: (department = 'Operations'::text)
+```
+
+For additional information, see [Create Index](https://docs.yugabyte.com/latest/api/ysql/the-sql-language/statements/ddl_create_index/#unique).
+
+## Removing Indexes
+
+You can remove one or more existing indexes using the `DROP INDEX` statement that has the following syntax:
+
+```
+DROP INDEX index_name1, index_name2, index_name3, ... ;
+```
+
+The following example shows how to remove `index_employees_department` that was created in [Creating Indexes](#creating-indexes):
+
+```sql
+DROP INDEX index_employees_department;
+```
+
+If you execute the same `SELECT` query with the `EXPLAIN` statement as in [Creating Indexes](#creating-indexes), the query plan will not include any information about the index. 
+
+## Listing Indexes
+
+YSQL inherits all the functionality of the PostgeSQL `pg_indexes` view that allows you to retrieve a list of all indexes in the database as well as detailed information about every index.
+
+For details, see [pg_indexes](https://www.postgresql.org/docs/12/view-pg-indexes.html) in the PostgreSQL documentation.
+
+## Using the UNIQUE Index
```

Comment:
I think "using *a* unique .."  or simply "Using unique indexes" matching the style for the other sections is better.
"the" might suggest there is only one unique index which is not the case.

Reply:
Fixed